Assets/Scripts/Enemies/SalivaBullet.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// BULLET FOR SPACE FLY (ADULT) !!!!!!
+/// </summary>
+
+public class SalivaBullet : Bullet
+{
+    [SerializeField]
+    private float bulletSpeed = 0.07f;
+
+    private bool reachedDest = false;
+    Vector3 targetPos;
+
+    void Start()
+    {
+
+        //face player
+        Vector3 dir = transform.TransformDirection(targetPos - transform.localPosition);
+        transform.rotation = Quaternion.LookRotation(dir);
+
+        timer = bulletDespawnTime;
+    }
+
+    void Update()
+    {
+        timer -= Time.deltaTime;
+        if (timer < 0) Destroy(transform.gameObject);
+
+        //move toward grabbed player location
+        transform.localPosition += transform.forward * bulletSpeed;
+    }
+
+    private void OnCollisionEnter(Collision collision)
```

Comment:
this migt be a method that could be removed once the bullet scripts are updated, for now its fine however